src/lib/duration/humanize.js

```diff
@@ -17,7 +17,7 @@ function substituteTimeAgo(string, number, withoutSuffix, isFuture, locale) {
 function relativeTime (posNegDuration, withoutSuffix, locale) {
     var duration = createDuration(posNegDuration).abs();
     var seconds  = round(duration.as('s'));
-    var minutes  = round(duration.as('m'));
+    var minutes  = round(duration.as('m')) || Math.ceil(duration.as('m'));
```

Comment:
Can you explain the problem in more detail? What is the problem if `duration.as('m') === 0`, or its not `0`, but falsey?